tools/openterm.cmd

```diff
@@ -0,0 +1,42 @@
+@echo off

```

Comment:
AH. I forgot this was in the PR. I'll move this to another branch, because this obviously doesn't work.